app/models/order.rb

```diff
@@ -267,6 +268,28 @@ def can_transition
     return (required_process_checks - process_checklists).empty?
   end
 
+  def send_submission_pickup_email?
+    booking_type.appointment? || order_transport&.pickup?
```

Comment:
Appointment and pickup have the same email ? 🤔

Reply:
yep, https://jira.crossroads.org.hk/browse/GCW-2605